pandas/io/tests/test_json/test_pandas.py

```diff
@@ -821,6 +821,96 @@ def my_handler_raises(obj):
                           DataFrame({'a': [1, 2, object()]}).to_json,
                           default_handler=my_handler_raises)
 
+    def test_categorical(self):
+        df = DataFrame({"A": ["a", "b", "c", "a", "b", "b", "a"]})
+        df["B"] = df["A"]
+        expected = df.to_json()
+
+        df["B"] = df["A"].astype('category')
+        self.assertEqual(expected, df.to_json())
+
+        s = df["A"]
+        sc = df["B"]
+        self.assertEqual(s.to_json(), sc.to_json())
+
+    def test_datetime_tz(self):
+        tz_range = pd.date_range('20130101', periods=3, tz='US/Eastern')
```

Comment:
ideally put the issue number associated (for those tests which you added and its appropriate)